util/reggen/uvm_reg.sv.tpl

```diff
@@ -235,11 +235,18 @@ package ${block.name.lower()}_ral_pkg;
   % for b in block.blocks:
     % for base_addr, inst, _ in block.block_name_to_instances[b.name]:
       ${inst} = ${gen_dv.bcname(b)}::type_id::create("${inst}");
+<%
```

Comment:
@weicaiyang if you look at top_earlgrey.hjson, the ast module has a special attribute called `reggen_only`.
if you look [here](https://github.com/lowRISC/opentitan/blob/master/hw/top_earlgrey/data/top_earlgrey.hjson#L223), it gives a rough idea of the differences.

Can you use this as a signal to handle certain modules differently? We can give them better attribute names if that helps.

Reply:
Thanks @tjaychen
I updated to use `attr` to decide the hier path.